src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readFromFile(fromFileName)));
+    }
+
+    private int[] readFromFile(String fromFileName) {
+        try (BufferedReader inputFile = new BufferedReader(new FileReader(fromFileName))) {
+            final String coma_separator = ",";
+            final String supply_word = "supply";
+            final String buy_word = "buy";
+            final int operation_type_index = 0;
+            final int amount_index = 1;
+            int supplyTotal = 0;
+            int buyTotal = 0;
```

Comment:
method `readFromFile()` must not make any calculations. it should read lines from the file put them to array of `String`, where each element is separate line from the file, and just return this array.
create a method `calculateStatistic()` which will take the array of String and move logic  connected  to `supplyTotal`  and `buyTotal` to this method.